### ISSUE TITLE
Page section missing lang attribute

### ISSUE THREAD

**Issue by GGD4V** [MeD-DMC/adapt_authoring_college#10]
https://esdc.prv/cgi-bin/scdtic-ictdcs/page-error-view.aspx?id=15277&wid=0

**Comment by ftourangeau** [comment]
Fixed
